app/styles/main.scss

```diff
@@ -197,13 +197,311 @@ a:hover, a:focus {
 .main-search-template {
     margin: 3px 5px;
     padding: 3px 5px;
-    .alias {
-        font-size: 13px;
+    line-height: 25px;
+    .main-search-alias, .main-search-annotation {
         color: grey;
-        font-style: italic;
-        font-weight: 200;
+        font-size: 13px;
         line-height: 15px;
     }
+    .main-search-alias {
+        font-weight: 300;
+        font-style: italic;
+    }
+    .main-search-variant {
+        .match {
+            margin-bottom: 3px;
+            width: 100%;
+            height: 22px;
+            .main-search-gene-variant {
+                float: left;
+                height: 16px;
+                line-height: 16px;
+                margin-top: 6px;
+            }
+            .oncogenic-icon-image {
+                float: left;
+                margin-left: 5px;
+                width: 24px;
+                height: 22px;
+                line-height: 22px;
+                background-image: url("../resources/images/oncogenic_v2_09302016.png");
+                -webkit-background-size: 200px;
+                background-size: 200px;
+                display: block !important;
+            }
+            .no-annotation {
+                float: right
+            }
+        }
+
+        .oncogenic-icon-image.level1.oncogenic {
+            background-position: -13px -14px;
+        }
+
+        .oncogenic-icon-image.level2A.oncogenic {
```

Comment:
Could we assign the background-position systematically instead of manually like this? like using a for loop? 
It is hard to manage this way.